src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -62,7 +65,7 @@ class ProjectCreationModel {
     fun doOnUserSelection(selectedCollection: Collection, workspace: Workspace) {
         if (selectedCollection.labelKey == "book") {
             createProject(selectedCollection)
-            workspace.dock<ProjectHomeView>()
+            showOverlayProperty.set(true)
```

Comment:
I think it might be better to insert this between lines 102 & 103 just inside in the null check. In the off chance something were to happen and the language was null, the overlay would get stuck on the screen.